src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothData.cs

```diff
@@ -542,4 +542,154 @@ public string ServiceUuid
+        public int AxisY
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// The padding value, -128 ~127
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int Padding
+        {
+            get;
+            set;
+        }
+    }
+
+    /// <summary>
+    /// This class contains the HID keyboard event information.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BluetoothHidKeyData
+    {
+        /// <summary>
+        /// The modifier keys : such as shift, alt
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public byte Modifier
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// The key value - currently pressed keys : Max 8 at once
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public byte[] Key
+        {
+            get;
+            set;
+        }
+    }
+
+    /// <summary>
+    /// This class contains the data received from the HID Host.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BluetoothHidDeviceReceivedData
+    {
+        /// <summary>
+        /// The default constructor. Initializes an object of the BluetoothHidReceivedData.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        internal static BluetoothHidDeviceReceivedData Create(BluetoothHidDeviceReceivedDataStruct structInfo)
+        {
+            BluetoothHidDeviceReceivedData receivedData = new BluetoothHidDeviceReceivedData();
+            receivedData.Address = structInfo.RemoteAddress;
+            receivedData.HeaderType = structInfo.headerType;
+            receivedData.ParamType = structInfo.paramType;
+            if (structInfo.dataSize > 0)
+            {
+                receivedData.DataLength = structInfo.dataSize;
+                receivedData.Data = new byte[receivedData.DataLength];
+                Marshal.Copy(structInfo.data, receivedData.Data, 0, receivedData.DataLength);
+            }
+            return receivedData;
+        }
+
+        /// <summary>
+        /// The remote device's address
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Address
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// The header type
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BluetoothHidHeaderType HeaderType
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// The parameter type
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BluetoothHidParamType ParamType
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// The received data length
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int DataLength
```

Comment:
No need, Becuase, Data.Length was provided

Reply:
Done. Thank you.